bundle.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
```

Comment:
shouldn't this file be called `Dockerfile.bundle`? (all the other docker files start with Dockerfile)

Reply:
bundle.Dockerfile is auto generated from "operator-sdk generate bundle -q --overwrite --version $(OPERATOR_VERSION)" in the operator-bundle target. I'll remove it from the repo, as it's an artifact from the generate bundle process.